logging-toolkit/src/lib.rs

```diff
@@ -11,11 +13,12 @@ use slog::LevelFilter;
 use slog::Logger;
 use std::env;
 
-pub fn make_logger(
-    root_name: &'static str,
-    use_json_env_name: &str,
-    min_log_level_env_name: &str,
-) -> Logger {
+lazy_static! {
+    static ref ROOT_LOGGER: Logger =
+        make_root_logger("RUST_PROOFS_LOG_JSON", "RUST_PROOFS_MIN_LOG_LEVEL");
```

Comment:
Should probably be `RUST_FIL_PROOFS…` now.

Reply:
Maybe just `FIL_PROOFS`? it really doesn't matter that this is written in rust for env variables